### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jakubMikolajczykTech**
[DEMO LINK](https://kabaktech.github.io/layout_calendar/)

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,53 @@
-body {
-  margin: 0;
+.calendar {
+  width: 707px;
+  margin: 0 auto;
```
Comment (mvjl000) [inline src/styles/main.scss:3]: The calendar should be centered both horizontally and vertically. Maybe try to add display flex to parent, which is body.

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,53 @@
-body {
-  margin: 0;
+.calendar {
```
Comment (mvjl000) [inline src/styles/main.scss:1]: `.calendar {}` should not be nested inside of body. Please change it

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,53 @@
...
+  width: 707px;
```
Comment (mvjl000) [inline src/styles/main.scss:2]: Avoid such a hard-coded values.
This is like a magic number, it is not clear why is it set to 707px and if you change for instance dimensions of calendar-day things will break up.
You should calculate the width basing on your variables ;)

Please create a separate file for variables

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,53 @@
...
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 30px 10px 10px;
```
Comment (mvjl000) [inline src/styles/main.scss:7]: You won't need this paddings after centering calendar inside of body.

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,53 @@
...
+
+  $days: mon, tue, wed, thu, fri, sat, sun;
```
Comment (mvjl000) [inline src/styles/main.scss:9]: This should also go to variables file I mentioned before

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,53 @@
...
+      font-family: Arial, sans-serif;
+      font-size: 30px;
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    background-color: #eee;
+    width: 100px;
+    height: 100px;
```
Comment (mvjl000) [inline src/styles/main.scss:39]: You need to create variables for day dimensions. Then you will be able to calculate evertything basing on them.

File: src/styles/main.scss [hunk 7]
```diff
@@ -1,3 +1,53 @@
...
+    border: 1px solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: transform 0.5s;
+
+    &:first-child:hover,
+    &:nth-child(n +2):hover {
```
Comment (mvjl000) [inline src/styles/main.scss:47]: I don't get those selectors, it's overcomplicated. Instead just do:
```suggestion
&:hover {
```

File: src/styles/main.scss [hunk 8]
```diff
@@ -1,3 +1,53 @@
...
+      background-color: pink;
+      cursor: pointer;
```
Comment (mvjl000) [inline src/styles/main.scss:49]: I would move `cursor: pointer;` up so it's inside of __day

File: src/styles/main.scss [hunk 9]
```diff
@@ -1,3 +1,53 @@
...
+  &__day {
+    box-sizing: border-box;
+    background-color: #eee;
+    width: 100px;
+    height: 100px;
+    border: 1px solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: transform 0.5s;
```
Comment (mvjl000) [inline src/styles/main.scss:44]: List all properties for transition (but don't use `all` either). Background-color is not listed and thus, it's not animated - it turns pink immediately

**Review by mvjl000 — CHANGES_REQUESTED** [review]

**Review by choeqq — CHANGES_REQUESTED** [review]
Waiting for any updates @kabakTech 👀

**Pull request closed without merging**